admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/screen/dukan_details/DukanDetailsMapper.kt

```diff
@@ -0,0 +1,21 @@
+package net.thechance.mena.admin_panel.presentation.screen.dukan_details
+
+import net.thechance.mena.admin_panel.domain.entity.dukan.Dukan
+import net.thechance.mena.admin_panel.domain.entity.dukan.Dukan.ActivationStatus
+import kotlin.uuid.ExperimentalUuidApi
+
+@OptIn(ExperimentalUuidApi::class)
+fun Dukan.toUi() = DukanDetailsScreenState.DukanUi(
```

Comment:
just for consistency
```suggestion
fun Dukan.toUiState() = DukanDetailsScreenState.DukanUi(
```